src/legacy/server/http/integration_tests/default_route_provider.test.ts

```diff
@@ -29,7 +29,7 @@ let mockDefaultRouteSetting: any = '';
 describe('default route provider', () => {
   let root: Root;
   beforeAll(async () => {
-    root = kbnTestServer.createRoot();
+    root = kbnTestServer.createRoot({ migrations: { skip: true } });
```

Comment:
NIT: It seems you adapted every call to `createRoot` to add this. Should we set `migrations: { skip: true }` as a default in `kbnTestServer.createRoot` ?

Reply:
Some tests run with esArchiver and then they need to apply migrations. Ideally we shouldn't disable migrations, but we should disable the ES version check itself. There is `elasticsearch.ignoreVersionMismatch` but it's only available in development and our integration tests run against production. We could just make this option available in production, but I think it warrants a bigger discussion so I created  https://github.com/elastic/kibana/issues/56505